core-java/src/main/java/com/baeldung/ExceptionHandling/Exceptions.java

```diff
@@ -0,0 +1,251 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class Exceptions {
+	
+	 private final static Logger logger = Logger.getLogger("ExceptionLogging");

```

Comment:
@DavidLandup there are a few whitespacing problems here in the file. Take a minute to go through each of these and make sure that the file is pristine. I'll mark a couple of them, but please go through the file and repair all of them.

Reply:
Here is one. This field needs to be indented by 4 spaces.